crates/sui-json-rpc/src/api.rs

```diff
@@ -15,8 +15,15 @@ use sui_json_rpc_types::{
 use sui_open_rpc_macros::open_rpc;
 use sui_types::base_types::{ObjectID, SuiAddress, TransactionDigest};
 use sui_types::crypto::SignatureScheme;
+use sui_types::object::Owner;
 use sui_types::sui_serde::Base64;
 
+/// Maximum number of events returned in an event query.
+/// This is equivalent to EVENT_STORE_QUERY_MAX_LIMIT in `sui-storage` crate.
+/// To avoid unnecessary dependency on that crate, we have a reference here
+/// for document purposes.
+pub const EVENT_QUERY_MAX_LIMIT: usize = 100;
```

Comment:
yeah ... let's start small and test how much load it may expose to node. Maybe it's a bit too conservative though. But 5000 (the original numbers) seems too risky to support before we have a better idea